src/include/duckdb/function/scalar/strftime.hpp

```diff
@@ -74,7 +74,7 @@ struct StrTimeFormat {
 	//! Format is literals[0], specifiers[0], literals[1], ..., specifiers[n - 1], literals[n]
 	vector<string> literals;
 	//! The constant size that appears in the format string
-	idx_t constant_size;
+	idx_t constant_size = 0;
```

Comment:
Our compiler warnings didn't catch this?

Reply:
No, although this was not the problem/this was not used without uninitialization anyway. I mostly added this because I noticed in a debugger it held garbage data.